Scoop.js

```diff
@@ -690,8 +690,13 @@ export class Scoop {
       this.targetUrlContentType = contentType
     }
 
-    // If text/html, bail from non-web content capture process.
+    // If text/html or no content-type, bail from non-web content capture process.
     // Scoop.capture will go based on the value of `this.targeredUrlIsWebPage`.
```

Comment:
```suggestion
    // Scoop.capture will go based on the value of `this.targetUrlIsWebPage`.
```
(This is not part of the PR, but I just noticed this typo.)